src/util/kaldi-table-inl.h

```diff
@@ -88,8 +101,10 @@ template<class Holder>  class SequentialTableReaderScriptImpl:
 
   SequentialTableReaderScriptImpl(): state_(kUninitialized) { }
 
+  // you may call Open from states kUninitialized and kError.
```

Comment:
I'll bear that in mind in future.   If I update this patch for something else I'll fix that.